tests/virt_autotest/set_config_as_glue.pm

```diff
@@ -0,0 +1,45 @@
+# SUSE's openQA tests
+#
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: set configuration as glue
+# Maintainer: Leon Guo <xguo@suse.com>
+
+package set_config_as_glue;
+
+use base "consoletest";
+use xen;
+use strict;
+use warnings;
+use testapi;
+
+sub virt_guests {
+    my $wait_script        = "30";
```

Comment:
Can you make the name more like a function, eg fufill_guests_in_setting, or whatever you prefer at least as a operation?